tests/integration-tests/tests/performance_tests/test_openfoam/test_openfoam/pcluster.config.yaml

```diff
@@ -13,6 +12,7 @@ HeadNode:
   Iam:
     AdditionalIamPolicies:
       - Policy: arn:aws:iam::aws:policy/AmazonSSMManagedInstanceCore # Required to report patching status
+      - Policy: arn:aws:iam::aws:policy/AmazonS3ReadOnlyAccess #Required to read software artifacts from s3
```

Comment:
Let's be more specific and add the permission to the sopecific folder containing the OpenFOAM object.

```
HeadNode:
  Iam:
    S3Access:
      - BucketName: performance-tests-resources-for-parallelcluster
        EnableWriteAccess: false
```